lib/htmlbuilder.js

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env node
+const fs = require('fs');
+const Handlebars = require('handlebars');
+
+module.exports.buildHtml = (auditData) => {
+    var template = fs.readFileSync('../lib/template.html', 'utf8');
```

Comment:
Try `const template = require('template.html')` above this function declaration under `const Handlebars`

Reply:
Never mind, this doesn't.